jobsdb/jobsdb.go

```diff
@@ -497,6 +497,24 @@ type HandleT struct {
 		mu      sync.Mutex
 		started bool
 	}
+
+	configGetter *config.Config
+	config       struct {
```

Comment:
there are various other fields in the Handle struct which would qualify as config fields

Reply:
have added them all and segregated things(`Handle.migrationConfig` and `Handle.backupConfig`).
I think there can be some more cleanup but right now I don't see it.